neo/rawio/intanrawio.py

```diff
@@ -487,7 +487,8 @@ def _get_analogsignal_chunk_one_file_per_signal(self, i_start, i_stop, stream_in
 
     def _demultiplex_digital_data(self, raw_digital_data, channel_ids, i_start, i_stop):
 
-        output = np.zeros((i_stop - i_start, len(channel_ids)), dtype=np.uint8)
+        dtype = np.uint16 # We fix this to match the memmap dtype
```

Comment:
This is the only real change on this PR. All the other is a variable re-name to avoid further confusion.